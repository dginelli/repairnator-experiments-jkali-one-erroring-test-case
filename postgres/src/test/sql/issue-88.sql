SELECT * FROM zdb_range_agg('so_posts', 'favorite_count', '[{"from":10},{"key":"bob","from":100,"to":200},{"to":1000}]', '') ORDER BY doc_count DESC;
