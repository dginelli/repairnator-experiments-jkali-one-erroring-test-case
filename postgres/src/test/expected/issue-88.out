SELECT * FROM zdb_range_agg('so_posts', 'favorite_count', '[{"from":10},{"key":"bob","from":100,"to":200},{"to":1000}]', '') ORDER BY doc_count DESC;
   key    | low | high | doc_count 
----------+-----+------+-----------
 *-1000.0 |     | 1000 |     20845
 10.0-*   |  10 |      |      1540
 bob      | 100 |  200 |        29
(3 rows)

